dotcom-rendering/src/components/EuropeLandingModal.importable.tsx

```diff
@@ -418,3 +419,13 @@ export const EuropeLandingModal = ({ edition }: Props) => {
 		</dialog>
 	);
 };
+
+/**
+ * This is a weird way of handling this client-only island,
+ * but as this component is temporary, it was deemed
+ * a more pragmatic approach over having to rewrite it entirely.
+ *
+ * (October 2023)
+ */
+export const EuropeLandingModal = (props: Props) =>
+	isServer ? null : <ClientOnlyEuropeLandingModal {...props} />;
```

Comment:
Discussed with @oliverabrahams & @OllysCoding – this Island cannot run on the server.